### ISSUE TITLE
Cannot choose or change storage path on Windows 10, it only allows "/"

### ISSUE THREAD

**Issue by claude-cp** [BoostIO/Boostnote#3438]
## Current behavior
Cannot change local directory for notes, it only gives me the option to use the default "/". Please see screenshot

## Expected behavior
To be able to change the storage path of the main directory where it should keep the notes, either by choosing an existing folder, or even typing it manually or something, but there's nothing on the UI at the moment.

### First approach
1. Create a local "Storage" and give it a name
2. Go to general settings
3. Go to "Import"
4. In the "Folder" dropdown, you can only select "/" and there's no button anywhere to change this path, nor can you type manually.

### Second approach:
1. Create a local "Storage" and give it a name
2. Granted that you are on the "home" view of the app
3. Select the storage
4. While hovering on the name of *that* storage, click on the cog wheel for it
5. It takes you to a view where it says "Edit Storage" and you can only change the storage name, or delete the storage, but cannot change the path here either.

## Environment
- OS Version and name : Windows 10

**Comment by zzxxhhzxh** [comment]
Yes, I put all notes in OneDrive, now it's difficult to setup in new version

**Comment by Flexo013** [comment]
What version of Boostnote are you using?

**Comment by claude-cp** [comment]
I am not sure as I tried to find that (+ I am at work and couldn't check on windows now), but it was the one available for download from github on 16th of Jan 2020 :)

**Comment by claude-cp** [comment]
Guys, please do not take me posting the following link it in any bad respect, I just want to point out that I do not seem to be the only one who feels... cheated by the new version. I mean, it really used to work: https://www.reddit.com/r/Boostnote/comments/efgtz2/wtf_new_version_sucks_cant_select_local_storage/

Just now, a friend of mine asked me out of the blue what happened with the new version too, so it figures ..

**Comment by Flexo013** [comment]
Don't worry, I understand you completely. I still use the old app as well, since the new app does not work for me in its current state.

You can go here to get the latest version: https://github.com/BoostIO/boost-releases/releases/

As for the initial issue I think we can close this as it only pertains to Boost Note.next

**Comment by claude-cp** [comment]
> As for the initial issue I think we can close this as it only pertains to Boost Note.next

I'll pick the older version when I go home too. 
However, I don't agree we can "close this". Even if it's relevant for one of the versions, it's the version that matters. It's an issue that we shouldn't sweep under the rug, *especially* going forward.

**Comment by Flexo013** [comment]
Boost Note.next has it's own repo. I'll transfer this issue.
